### ISSUE TITLE
incorrect pallet path in tutorial

### ISSUE THREAD

**Issue by TovarishFin** [substrate-developer-hub/substrate-developer-hub.github.io#522]
at https://github.com/substrate-developer-hub/substrate-developer-hub.github.io/blob/source/docs/tutorials/creating-a-runtime-module.md, there is the following code shown:
```
# --snip--

[dependencies.test-pallet]
default-features = false
path = '../pallets/template'

# toward the bottom
[features]
default = ['std']
std = [
    'test-pallet/std',
    # --snip--
]
```
should this not be as follows instead?
```
# --snip--

[dependencies.test-pallet]
default-features = false
path = '../pallets/my-template'

# toward the bottom
[features]
default = ['std']
std = [
    'test-pallet/std',
    # --snip--
]
```
If using the original block of code, it would just use the already included pallet template rather than the pallet that is cloned earlier in the tutorial (at least this is how I understood it).

**Comment by JoshOrndorff** [comment]
Hello @TovarishFin Thanks for this issue.

I've looked at the file you linked, but I do not see the snippet you quoted.

**Comment by TovarishFin** [comment]
Sorry about that. I clicked the edit button in the tutorial and it took me to that file... not sure how the tutorials are built and deployed but you can find the block mentioned above in the actual tutorial link (section: Add Your Pallet to Your Node):
https://substrate.dev/docs/en/tutorials/creating-a-runtime-module/.
I assumed that the markdown file would match the tutorial.

**Comment by JoshOrndorff** [comment]
Okay got it. Thanks for reporting. I've fixed this in #523.
